_posts/2017-09-02-merger.md

```diff
@@ -0,0 +1,74 @@
+
+On August 30, 2017, the Software Carpentry and Data Carpentry Steering Committees met jointly and approved the following two motions, which together form a strong commitment to continue moving forward with the merger, and to eventually hand off governance to a joint Carpentries Steering Committee:
+- Approve merger of Software Carpentry and Data Carpentry: The Software Carpentry and Data Carpentry steering committees approve the merger of the two organizations into a single umbrella organization with associated lesson organizations, with a starting date of January 1, 2018.
+- Approve appointed members of combined board: We appoint Karen Cranston, Kate Hertweck, Mateusz Kuzak, Sue McClatchy, and Ethan White to the board of the umbrella organization.
+
+We are now very excited for the next steps in putting together the Carpentries! Even though the two motions passed this week lay the groundwork for a merged organization, there’s still a lot of work to be done on the details of how this will all come together. The two Steering Committees (which remain in command until Dec 31) will be putting together Requests for Comment (https://www.mediawiki.org/wiki/Requests_for_comment), because community input in decisions and structure around everything from governance to curriculum oversight will be key.
+
+#### A very brief history of Software and Data Carpentry
+
+![History](/images/SWCDChistory.png "Milestones in the history of Software Carpentry and Data Carpentry")
+
+## What does this mean for our instructors?
+
+Software and Data Carpentry have already unified their instructor training to have one Carpentries instructor certification and program. Hence, the instructor training program will continue as it is. People who are already instructors can continue to teach workshops as they already do! Nothing substantive will change. There may be some updates on email list locations, but it will remain the joint Carpentries instructor community that it already is. 
+
+The new Carpentries Steering Committee will include elected positions, and instructors will be the electorate casting votes for those, as they have in the past for the Software Carpentry Steering Committee. More information about the elections will be coming out in October. Please also consider running yourself for a position on the Steering Committee to help guiding the Carpentries through this next phase! If you’re interested in more information on elections now, please contact Kate Hertweck (k8hertweck@gmail.com). 
+
```

Comment:
## What does this mean for our Member Organizations?

Memberships are already joint between Software and Data Carpentry, so there will be no changes with memberships. All signed, pending and upcoming membership agreements will remain valid and terms will continue to be the same with the Carpentries as the are with Software and Data Carpentry. We will be transitioning the Software Carpentry Advisory Council to a joint Carpentries council, likely in January, and will keep members updated on that transition.